libraries/PDM/src/PDM.cpp

```diff
@@ -203,13 +214,24 @@ void PDMClass::IrqHandler()
   }
 }
 
-extern "C" {
-  void PDM_IRQHandler(void);
-}
-
-void PDM_IRQHandler(void)
+extern "C"
 {
-  PDM.IrqHandler();
+  void PDM_IRQHandler(void)
+  {
+    PDM.IrqHandler();
+  }
 }
 
-extern PDMClass PDM;
+#ifndef PIN_PDM_DIN
+#define PIN_PDM_DIN -1
+#endif
+
+#ifndef PIN_PDM_CLK
+#define PIN_PDM_CLK -1
+#endif
+
+#ifndef PIN_PDM_PWR
+#define PIN_PDM_PWR -1
+#endif
+
+PDMClass PDM(PIN_PDM_DIN, PIN_PDM_CLK, PIN_PDM_PWR);
```

Comment:
PDM is declared within the class like Serial or Wire. It is consistent with current usage of PDM from Arduino nRF528x-mbedos. Making using 3rd party library easier. 

Note: it requires to remove this line from CPlay library https://github.com/adafruit/Adafruit_CircuitPlayground/blob/master/utility/Adafruit_CPlay_Mic.cpp#L30 . I will submit an PR for this after this once is merged.